sepal_ui/planetapi/planet_view.py

```diff
@@ -0,0 +1,120 @@
+import sepal_ui.scripts.utils as su
+from traitlets import observe, dlink, Bool
+from planet_model import PlanetModel
+import ipyvuetify as v
+import sepal_ui.sepalwidgets as sw
+
+
+class StateIcon(sw.Tooltip):
```

Comment:
Don't you think this very one could be useful as a general component in sw ?

could you also add a super simple test where you just init the component (so that codecov stops complaining)

Reply:
I've tested on my side in a fresh env and it worked as it was working before, just `command`, so I think that entry point is managing this... 
Now that they are python scripts, I think we can make specific tests, and validate them by capturing their `stdout`, what do you think?

Reply:
I think you answered in the wrong PR ;-)